implementation/src/main/java/io/smallrye/mutiny/groups/MultiCollect.java

```diff
@@ -164,6 +165,28 @@ public MultiCollect(Multi<T> upstream) {
         return collector(upstream, Collectors.toMap(actualKM, actualVM), false);
     }
 
+    /**
+     *
+     * @param keyMapper a {@link Function} to map item to a key for the {@link Map}. Must not be {@code null},
+     *        must not produce {@code null}
+     * @param valueMapper a {@link Function} to map item to a value for the {@link Map}. Must not be {@code null},
+     *        must not produce {@code null}
+     * @param mergeFunction a {@link BinaryOperator} used to resolve collisions between values associated with the same key
```

Comment:
Must not be `null`, must not return `null`:
- this should be documented as for the other 2 parameters, and
- this should be tested